packages/creator-hub/main/vite.config.js

```diff
@@ -16,7 +16,7 @@ const config = {
   resolve: {
     alias: {
       '/@/': join(PACKAGE_ROOT, 'src') + '/',
-      '/shared/': join(PROJECT_ROOT, 'packages', 'shared') + '/',
+      '/shared/': join(PACKAGE_ROOT, '../shared') + '/',
```

Comment:
I think using:
```suggestion
      '/shared/': join(PROJECT_ROOT, 'shared') + '/',
```
should work